types/google-apps-script/apis/drive_v3.d.ts

```diff
@@ -1143,6 +1143,7 @@ declare namespace GoogleAppsScript {
                             resource: Drive.V3.Schema.Reply,
                             fileId: string,
                             commentId: string,
+                            optionalArgs: { fields: string },
```

Comment:
optionalArgs... should have been marked optional.

Reply:
It is named "optionalArgs" and yes they are optional, you can omit them and the call still works.

Reply:
No, for these 2 particular API calls the call errors out without fields...

Reply:
Using Apps Scripts just now: 

<img width="983" alt="Screenshot 2025-02-25 at 4 09 01 PM" src="https://github.com/user-attachments/assets/eb35e0ce-2776-4e3b-abc2-556d2a3be07a" />

However I do see the failure you mention when executed. Odd.

Reply:
Yeah, as written in the PR body, the tooltip in Script editor is incorrect here.